### ISSUE TITLE
Cannot published gem using "rake release"

### ISSUE THREAD

**Issue by dentarg** [twingly/twingly-url#139]
Same as https://github.com/twingly/capistrano-twingly/issues/48, but adding an issue here for visibility 

Trying to release with the rake task just hangs, the output about the error came after I hit enter a few times:

```
$ bundle exec rake release
twingly-url 6.0.2 built to pkg/twingly-url-6.0.2.gem.
Tagged v6.0.2.
Pushed git commits and tags.


rake aborted!
Pushing gem to https://rubygems.org...
You have enabled multi-factor authentication. Please enter OTP code.
Code:   You have enabled multifactor authentication but no OTP code provided. Please fill it and retry.
/Users/dentarg/.gem/ruby/2.6.3/bin/bundle:23:in `load'
/Users/dentarg/.gem/ruby/2.6.3/bin/bundle:23:in `<main>'
Tasks: TOP => release => release:rubygem_push
(See full trace by running task with --trace)
```

I completed the release by doing

```
$ gem push pkg/twingly-url-6.0.2.gem
Pushing gem to https://rubygems.org...
You have enabled multi-factor authentication. Please enter OTP code.
Code:   414317
Successfully registered gem: twingly-url (6.0.2)
```

**Comment by dentarg** [comment]
It [looks like](https://github.com/bundler/bundler/commit/1b2bbc73642c100749831f30d8a2f0e674c2f944) the fix in bundler is in v2.1.0.pre.1.

Time to try to go to Bundler 2 I guess. I'm still using:

```
$ bundle --version
Bundler version 1.17.3
```

**Comment by dentarg** [comment]
I changed our [MFA settings](https://guides.rubygems.org/setting-up-multifactor-authentication/) to "UI only" (needed for #135 anyway), so we can close this.
